configuration/cdap-env.xml

```diff
@@ -20,7 +20,7 @@
 
   <property>
     <name>apt_repo_url</name>
-    <value>http://repository.cask.co/ubuntu/precise/amd64/cdap/3.4</value>
+    <value>http://repository.cask.co/ubuntu/precise/amd64/cdap/3.6</value>
```

Comment:
Not sure what this value should be. I would have expected either 3.5 (because that's been released) or 4.0 (because that is what the develop branch is building), but I'll trust that this is correct.